micro-rdk-ffi/build.rs

```diff
@@ -20,9 +20,6 @@ fn main() {
     #[cfg(not(feature = "test"))]
     {
         if env::var("TARGET").unwrap() == "xtensa-esp32-espidf" {
-            if std::env::var_os("IDF_PATH").is_none() {
```

Comment:
A lack of `IDF_PATH` in the environment is no longer indicative of a misconfigured environment.